plugins/polio/tests/test_base.py

```diff
@@ -683,6 +686,181 @@ def test_create_calendar_xlsx_sheet_with_separate_scopes_per_round(self):
         self.assertEqual(data_dict["January"][0], self.format_date_to_test(c, c_round_1))
         self.assertEqual(data_dict["January"][1], self.format_date_to_test(c, c_round_2))
 
+    @staticmethod
+    def campaign_csv_columns():
+        return [
+            "country",
+            "obr_name",
+            "vaccine_types",
+            "onset_date",
+            "pv_notified_at",
+            "round_number",
+            "round_start_date",
+            "round_end_date",
+            "ra_submission_date",
+            "ra_approval_date",
+            "budget_submitted_date",
+            "budget_approved_date",
+            "who_disbursed_to_co_at",
+            "who_disbursed_to_moh_at",
+            "unicef_disbursed_to_co_at",
+            "unicef_disbursed_to_moh_at",
+            "gpei_coordinator",
+            "round_target_population",
+            "doses_requested",
+            "cost",
+            "lqas_district_passing",
+            "lqas_district_failing",
+            "preparedness_spreadsheet_url",
+            "preparedness_sync_status",
+        ]
+
+    @staticmethod
+    def create_org_unit(id, name, type, version):
+        return OrgUnit.objects.create(
+            id=id,
+            name=name,
+            org_unit_type=type,
+            version=version,
+        )
+
+    @staticmethod
+    def row_data(country, obr_name, round_number, round_started_date, round_ended_date):
+        return [
+            country,
+            obr_name,
+            "",
+            "",
+            "",
+            str(round_number),
+            round_started_date,
+            round_ended_date,
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
+            "",
```

Comment:
A lot of CSV columns are empty.

We should ensure that all the expected data is available, otherwise we won't know if something is missing or broken.